src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -410,18 +454,31 @@ export const TradeInput = memo(() => {
         )}
 
         <ManualAddressEntry />
+        {maybeUnsafeTradeWarning}
         <Tooltip label={activeQuoteStatus.error?.message ?? activeQuoteStatus.quoteErrors[0]}>
-          <Button
-            type='submit'
-            colorScheme={quoteHasError ? 'red' : 'blue'}
-            size='lg-multiline'
-            data-test='trade-form-preview-button'
-            isDisabled={shouldDisablePreviewButton}
-            isLoading={isLoading}
-            mx={-2}
-          >
-            <Text translation={activeQuoteStatus.quoteStatusTranslation} />
-          </Button>
+          {isUnsafeQuoteNoticeDismissed === false ? (
+            <Button
+              colorScheme='red'
+              size='lg-multiline'
+              mx={-2}
+              // eslint-disable-next-line react-memo/require-usememo
+              onClick={() => setIsUnsafeQuoteNoticeDismissed(true)}
+            >
+              <Text translation={'defi.modals.saversVaults.understand'} />
+            </Button>
+          ) : (
```

Comment:
can we update the tooltip on hover?

![Screenshot 2023-12-08 at 3 07 34 PM](https://github.com/shapeshift/web/assets/15096737/0edeee1e-a26a-4b49-9efe-d50a25b4b60f)

Reply:
Removed tooltips entirely since they're rugged for *all* errors, we should've never gotten a tooltip there in the first place unless we had meaningful error messages